recipes/noiseflow/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set org = "shakeflow" %}
+{% set name = "noiseflow" %}
+{% set version = "0.0.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 20736acea7dde31bbd3c6c6c3fbe922bedf242dd0b64bcd02a4557b4d566b427
+
+build:
+  number: 0
+  script: 
+    - git clone https://github.com/xtensor-stack/xtensor-fftw extern/xtensor-fftw  # [unix]
```

Comment:
```suggestion
    # to use the custom FindPackage CMake assistance
    - git clone https://github.com/xtensor-stack/xtensor-fftw extern/xtensor-fftw  # [unix]
```

Reply:
It not works when I use `find_package(fftw)` directly, I need to specify the `cmake module path` which is from `xtensor-fftw` package. And that is why I should use git clone for `xtensor-fftw`
```cmake
set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/extern/xtensor-fftw/cmake/Modules/findFFTW") # use FindFFTW.cmake to find fftw
find_package(FFTW REQUIRED COMPONENTS ${REQUIRE_FLOAT_LIB} ${REQUIRE_DOUBLE_LIB} ${REQUIRE_LONG_DOUBLE_LIB})
```